js/client/src/components/app/Sidebar.js

```diff
@@ -100,11 +100,13 @@ function ConnectedSideBar({
         </div>
         <div className="left">
           Type
-          <div className="leftData">{data.body_style}</div>
+          {/* If abbreviation is not in the table, use 'Other' */}
+          <div className="leftData">{T.typeTable[data.body_style] || "Other"}</div>
```

Comment:
I know you're just using the existing import, but not a fan of `T` as a variable name here. Would prefer something more descriptive like `tables` Not a dealbreaker though.